packages/assets-controllers/src/token-prices-service/codefi-v2.test.ts

```diff
@@ -58,7 +58,7 @@ describe('CodefiTokenPricesServiceV2', () => {
       });
     });
 
-    it('throws if one of the token addresses cannot be found in the response data', async () => {
+    it('should not throw if one of the token addresses cannot be found in the response data', async () => {
```

Comment:
That's true that it doesn't throw, but is there something else that it does instead? Perhaps we ought to be more descriptive about the desired behavior in this case.